internal/pkg/cli/app_init.go

```diff
@@ -260,6 +260,20 @@ func (opts *InitAppOpts) listDockerfileDirs() ([]string, error) {
 	return directories, nil
 }
 
+func (opts *InitAppOpts) listDockerfileSelections() ([]string, error) {
+	dockerfileDirs, err := opts.listDockerfileDirs()
+	if err != nil {
+		return nil, err
+	}
+	var dockerfiles []string
+	for _, dir := range dockerfileDirs {
+		fmt.Printf(dir)
+		file := dir + "/Dockerfile"
+		dockerfiles = append(dockerfiles, file)
```

Comment:
You might be able to simplify this by doing `append(dockerfiles, fmt.Sprintf("%s/Dockerfile")) `